app/presenters/monitoring-stations/view-monitoring-stations.presenter.js

```diff
@@ -0,0 +1,135 @@
+'use strict'
+
+/**
+ * Formats the monitoring station details and related licences data for the
+ * `/monitoring-stations/{monitoringStationId}` page
+ * @module ViewMonitoringStationsPresenter
+ */
+
+const { formatAbstractionPeriod, formatLongDate } = require('../base.presenter.js')
+
+/**
+ * Formats the monitoring station details and related licences data for the
+ * `/monitoring-stations/{monitoringStationId}` page
+ *
+ * @param {string} auth - The auth object taken from `request.auth` containing user details
+ * @param {GaugingStationModel[]} monitoringStation - The monitoring station and associated licences data returned
+ * by FetchMonitoringStations
+ *
+ * @returns {object} monitoring station and licence data needed by the view template
+ */
+function go (auth, monitoringStation) {
+  const formattedLicences = _formatLicences(monitoringStation.licenceGaugingStations)
+  const sortedLicences = _sortLicences(formattedLicences)
+  const groupedLicences = _groupLicences(sortedLicences)
+
+  return {
+    gridReference: monitoringStation.gridReference,
+    hasPermissionToManageLinks: _checkPermissions(auth, 'manage_gauging_station_licence_links'),
+    hasPermissionToSendAlerts: _checkPermissions(auth, 'hof_notifications'),
```

Comment:
It's a penchant of mine, but we try to avoid prefixing things with `has`, `is`, `get` or `set`.

Just need to drop it (not come up with an alternate prefix!)

Also, calling `checkPermissions()` could be dropped in favour of `auth.credentials.scope.includes('manage_gauging_station_licence_links')`